core/bases/src/main/java/com/chocolatecake/util/BindingAdapters.kt

```diff
@@ -145,6 +145,15 @@ fun <T> View.showWhenNoResult(list: List<T>?) {
     }
 }
 
+@BindingAdapter(value = ["app:hideWhenLoading"])
+fun View.hideWhenLoading(isLoading: Boolean?) {
+    if (isLoading == true) {
+        this.visibility = View.INVISIBLE
+    } else {
+        this.visibility = View.VISIBLE
+    }
+}
+
```

Comment:
use isVisible

Reply:
the logic of hideWhenLoading is the opposite of  isVisible